src/tests/QueryBuilderTests.cpp

```diff
@@ -216,6 +216,29 @@ TEST_CASE_METHOD(SqlTestFixture, "SqlQueryBuilder.FieldsWithBelongsTo", "[SqlQue
         });
 }
 
+TEST_CASE_METHOD(SqlTestFixture, "SqlQueryBuilder.ComplexOR", "[SqlQueryBuilder]")
+{
+    checkSqlQueryBuilder(
+        [](SqlQueryBuilder& q) {
+            using namespace std::string_view_literals;
+            return q.FromTable("BETON_SORTE")
+                .Select()
+                .LeftOuterJoin("BEN_DESIGN"sv, "SORTE_NR"sv, "NR"sv)
+                .RightOuterJoin("BETON_ZEMENTART"sv,
+                                [](SqlJoinConditionBuilder q) {
+                                    return q.On("NR", { .tableName = "BETON_SORTE", .columnName = "ZEMENTART_NR" })
+                                        .Or("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_2_NR" })
+                                        .Or("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_3_NR" })
+                                        .Or("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_4_NR" });
```

Comment:
```suggestion
                                        .OrOn("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_2_NR" })
                                        .OrOn("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_3_NR" })
                                        .OrOn("NR", { .tableName = "BETON_SORTE", .columnName = "ZEM_4_NR" });
```

Analogous to `Where` vs `OrWhere` - what do you two guys think about `On` vs `OrOn`?

Sounds more homogenous to me than `On` vs `Or`.

Reply:
will change